consensus-types/types/attestation_data_test.go

```diff
@@ -84,13 +85,13 @@ func TestAttestationData_MarshalSSZ_UnmarshalSSZ(t *testing.T) {
 			require.NoError(t, err)
 			require.NotNil(t, data)
 
-			var unmarshalled *types.AttestationData
```

Comment:
no need to instantiate a pointer before creating the object via `New...`. I love this